File: app/dbmodels/AssetDisk.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'AssetDisk',
    'menu'          => ''
);
$table['fields'] = array(
    'id'               => DBModels::type('increments', 
                                         array('visible' => false)),
    'asset_id'        => DBModels::type('integer', 
                                         array('visible' => false)),
    'name'             => DBModels::type('string'),
    'device'           => DBModels::type('string'),
    'mountpoint'       => DBModels::type('string'),
    'file_system_id'    => DBModels::type('integer'),
    'totalsize'        => DBModels::type('integer'),
    'freesize'         => DBModels::type('integer'),
    'is_deleted'       => DBModels::type('boolean'),
    'is_dynamic'       => DBModels::type('boolean'),
);

$table['relationships'] = array(
    'asset'     => array(
        'type'  => 'belongsTo',
        'item'  => 'Asset'),
    'filesystem'  => array(
        'type'  => 'belongsTo',
        'item'  => 'FileSystem'),
);

File: fusionglpi/database/asset_disk.py
from fusionglpi import db

class AssetDisk(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    asset_id = db.Column(db.Integer, db.ForeignKey('asset.id'))
    name = db.Column(db.String(255, None, True))
    device = db.Column(db.String(255, None, True))
    mountpoint = db.Column(db.String(255, None, True))
    filesystem_id = db.Column(db.Integer, db.ForeignKey('filesystem.id'))
    totalsize = db.Column(db.Integer, nullable=False, server_default="0")
    freesize = db.Column(db.Integer, nullable=False, server_default="0")
    is_deleted = db.Column(db.Boolean, nullable=False, server_default="0")
    is_dynamic = db.Column(db.Boolean, nullable=False, server_default="0")
